Resources/ServerInfo/SimpleStation14/Guidebook/Species/WizdenSpecies/Slime.xml

```diff
@@ -9,19 +9,28 @@ Slimes are sentient beings made from jelly that commonly appear as humans to ble
 
 ## Ability Differences
 
-- Slimes have jelly instead of blood, which is toxic to other species when consumed.
-- Slimes are incapable of living in cold environments, dying very quickly.
-- To contrast, Slimes are able to survive longer in hot environments.
-- Take less brute damage.
-- Take more slash damage.
-- Breathe Nitrogen instead of Oxygen.
+- Have jelly instead of blood, which is toxic to other species when consumed
+- Incapable of living in cold environments, dying very quickly
+- Able to live in warmer environments
+- Breathe Nitrogen instead of Oxygen
+- Poison damage is inverted
+- Take twice the blunt damage if more than 5
```

Comment:
Let's un-crunch this statement a little :P

Something along "Minor blunt impacts have no effect, but stronger ones tear them up"